cfme/tests/infrastructure/test_host_hostname.py

```diff
@@ -0,0 +1,31 @@
+
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon, toolbar
+from cfme.infrastructure.host import Host
+from cfme.web_ui import InfoBlock
+from cfme.fixtures import pytest_selenium as sel
+from cfme.configure.tasks import is_host_analysis_finished
+from utils.wait import wait_for
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = \
+        testgen.infra_providers(metafunc, required_fields=["ssh_credentials"])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_host_hostname(provider):
+    provider.load_details()
+    sel.click(InfoBlock.element("Relationships", "Nodes"))
```

Comment:
You can replace these two lines by:

```python
provider.summary.relationship.nodes.click()
```

Which will load the summary page and will allow you to click through.